src/test/java/com/puppycrawl/tools/checkstyle/utils/JavadocUtilsTest.java

```diff
@@ -337,4 +350,108 @@ public void testGetTokenNames() {
         assertEquals("Unexpected token name",
             "HTML_COMMENT", JavadocUtils.getTokenName(20073));
     }
+
+    //-@cs[JavaNCSS] High NCSS count due to the huge switch needed to test every case.
```

Comment:
Moving `switch` block to another method overshoots the _ncss_ count for that method. Another option would thus be to  split the switch block into two tests which seemed inappropriate to me.